frontend/src/components/CoursePage.js

```diff
@@ -1,360 +1,377 @@
-import React from 'react'
-import { Button, Stack, CardHeader, CardContent, Rating, Card, Typography, Grid, TextField, Paper, Box } from '@mui/material'
+import React from "react";
```

Comment:
This change feels unnecessary.

Reply:
Everything here is styling changes, not sure if there's something in particular you're pointing to.